abb_bringup/launch/abb_moveit.launch.py

```diff
@@ -32,94 +30,84 @@ def launch_setup(context, *args, **kwargs):
-        "move_group": {
-            "planning_plugin": "ompl_interface/OMPLPlanner",
-            "request_adapters": """default_planner_request_adapters/AddTimeOptimalParameterization default_planner_request_adapters/ResolveConstraintFrames default_planner_request_adapters/FixWorkspaceBounds default_planner_request_adapters/FixStartStateBounds default_planner_request_adapters/FixStartStateCollision default_planner_request_adapters/FixStartStatePathConstraints""",
-            "start_state_max_bounds_error": 0.1,
-        }
-    }
-    ompl_planning_yaml = load_yaml(
-        "abb_irb1200_5_90_moveit_config", "config/ompl_planning.yaml"
+        .robot_description_semantic(
+            file_path=os.path.join(
+                get_package_share_directory(
+                    f"{moveit_config_package.perform(context)}"
+                ),
+                "config",
+                f"{moveit_config_file.perform(context)}",
+            )
+        )
+        .planning_pipelines()
+        .robot_description_kinematics(
+            file_path=os.path.join(
+                get_package_share_directory(
+                    f"{moveit_config_package.perform(context)}"
+                ),
+                "config",
+                "kinematics.yaml",
+            )
+        )
+        .trajectory_execution(
+            file_path=os.path.join(
+                get_package_share_directory(
+                    f"{moveit_config_package.perform(context)}"
+                ),
+                "config",
+                "moveit_controllers.yaml",
+            )
+        )
+        .planning_scene_monitor(
+            publish_planning_scene=True,
+            publish_geometry_updates=True,
+            publish_state_updates=True,
+            publish_transforms_updates=True,
+            publish_robot_description=True,
+            publish_robot_description_semantic=True,
+        )
+        .joint_limits(
+            file_path=os.path.join(
+                get_package_share_directory(
+                    f"{moveit_config_package.perform(context)}"
+                ),
+                "config",
+                "joint_limits.yaml",
+            )
+        )
+        .to_moveit_configs()
     )
-    ompl_planning_pipeline_config["move_group"].update(ompl_planning_yaml)
 
-    # Trajectory Execution Functionality
-    moveit_simple_controllers_yaml = load_yaml(
-        "abb_irb1200_5_90_moveit_config", "config/moveit_controllers.yaml"
-    )
+    # MoveIt controllers
     moveit_controllers = {
-        "moveit_simple_controller_manager": moveit_simple_controllers_yaml,
+        "moveit_simple_controller_manager": load_yaml(
+            f"{moveit_config_package.perform(context)}",
+            "config/moveit_controllers.yaml",
+        ),
         "moveit_controller_manager": "moveit_simple_controller_manager/MoveItSimpleControllerManager",
     }
 
-    trajectory_execution = {
-        # MoveIt does not handle controller switching automatically
```

Comment:
This behavior was lost in the changes as by default, `moveit_manage_controllers` is set to `True` by the builder. I'll open a PR to update.